Source/Source.API.php

```diff
@@ -909,6 +909,9 @@ function save() {
 
 		$t_changeset_table = plugin_table( 'changeset', 'Source' );
 
+		# INEO SRO : ajout trace de mise au point FBR ?
+		error_log(print_r($this, TRUE));
```

Comment:
Is this debugging code ? If so, please remove it. If not, comment should be in English.